src/sqlfluff/core/rules/std/__init__.py

```diff
@@ -0,0 +1,17 @@
+# No docstring here as it would appear in the rules docs.
+# Rule definitions for the standard ruleset, dynamically imported from the directory.
+# noqa
+
+import os
+from glob import glob
+
+rules_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), "L*.py")
+
+__all__ = []
+
+for module in sorted(glob(rules_path)):
+    # Manipulate the module path to extract the filename without the .py
+    rule = module.split("/")[-1][:-3]
+    cmd = f"from sqlfluff.core.rules.std.{rule} import Rule_{rule}"
+    __all__.append(f"Rule_{rule}")
+    exec(cmd)
```

Comment:
This feels awful but not sure how else to do it. I tried playing with `importlib.import_module` but couldn't get the result I was after, as we still need to import the actual class objects.

Reply:
@barrywhart is this what you had in mind? Or something different?

Reply:
Update: https://github.com/sqlfluff/sqlfluff/pull/688#discussion_r559201536